### DISCUSSION TITLE
OCR B font missing

### DISCUSSION THREAD

**Discussion by rolfmblindgren** [Omikhleia/resilient.sile#9] [Q&A] [answered]
I can't find the source file that calls the OCR B font.  I have OCR B Std, but not OCR B.

**Comment by Omikhleia** [comment, marked as answer]
The "OCR B" font referred to in the [documentation](https://drive.google.com/file/d/1f54qDEGWaN-MFN932x-t0jm3V-5VSHee/view) (mentioned on the main README) and used by default is from https://tsukurimashou.osdn.jp/ocr.php.en

Would you want to use _another_ font, then as explained in the documentation, you may use the following command in your document:

```
\ean13:font[family=⟨family⟩]
```
"OCR B Std", if I remember correctly, is the name of the Adobe font (perhaps clear now for non-free use? I haven't checked recently). I don't have it installed, but I guess, if it has indeed this name, that the following should work.
```
\ean13:font[family=OCR B Std]
```

I am still working on improving the documentation, but I thought this part to be quite okay by now - If there's anything I could still improve there, comments are welcome!

(By the way, I guess that this issue/question could or should rather have been reported to the [barcode.sile](https://github.com/Omikhleia/barcodes.sile) repository. Well no problem anyway.)

**Reply by Omikhleia** [reply]
@rolfmblindgren By the way, I have opened the "discussions" section (I thought I had done it, actually) and moved your issue there.

**Reply by Omikhleia** [reply]
For any potential future reader: 
For convenience, Skala's OCR B font is now bundled in the **barcodes.sile** dependency (since [v1.1.0](https://github.com/Omikhleia/barcodes.sile/releases/tag/v1.1.0)), so users don't have to go through the trouble of finding it and installing it. It works out of the box.
Of course, the other discussions above regarding on how to use another font instead, would you wish to do so, are still valid.
